store/searchlayer/channel_layer.go

```diff
@@ -172,7 +170,7 @@ func (c *SearchChannelStore) searchAutocompleteChannels(engine searchengine.Sear
 	if len(channelIds) > 0 {
 		channels, err := c.ChannelStore.GetChannelsByIds(channelIds, includeDeleted)
 		if err != nil {
-			return nil, model.NewAppError("searchAutocompleteChannels", "app.channel.get_channels_by_ids.app_error", nil, err.Error(), http.StatusInternalServerError)
+			return nil, errors.Wrapf(err, "Failed to autocomplete channels in team")
```

Comment:
The error message should be different here. Sorry I thought that was implied. We can have something like "Failed to get channels by ids"

Reply:
sorry, i made the necessary changes